server/datastore/mysql/apple_mdm.go

```diff
@@ -2862,6 +2862,65 @@ func (ds *Datastore) GetMDMAppleDefaultSetupAssistant(ctx context.Context, teamI
 	return asst.ProfileUUID, asst.UploadedAt, nil
 }
 
+func (ds *Datastore) UpdateHostDEPAssignProfileResponses(ctx context.Context, payload *godep.ProfileResponse) error {
```

Comment:
apologies if you already know this. but just in case: we should also call this method to store the responses we get in: `server/worker/macos_setup_assistant.go`

Reply:
I had added those [here](https://github.com/fleetdm/fleet/pull/16973/files#diff-1092299ba8d3b798f70a35a3117d4318b263678f08f51ed3c50e3a476fd462bcR120) (but maybe you hadn't gotten to that part yet). Was there another place you were thinking of?

Reply:
ah, thanks! I think the link I clicked in Slack was for an old changeset? wasn't there, I swear!